tests/analyser/analyser.cpp

```diff
@@ -127,16 +113,12 @@ TEST(Analyser, undefinedVariables)
         "MathML ci element has the child text 'a' which does not correspond with any variable names present in component 'my_component'.",
         "MathML ci element has the child text 'b' which does not correspond with any variable names present in component 'my_component'.",
     };
-    const std::vector<libcellml::Issue::Cause> expectedCauses = {
```

Comment:
Why is this cause removed? Should be MATHML?

Reply:
You commented on something that is outdated...